clients/drcachesim/common/options.h

```diff
@@ -57,6 +57,7 @@
 // Constants used by specific tools.
 #define REPLACE_POLICY_NON_SPECIFIED ""
 #define REPLACE_POLICY_LRU "LRU"
+#define REPLACE_POLICY_BIT_PLRU "BIT_PLRU"
```

Comment:
op_TLB_replace_policy in options.cpp lists the supported policies (aside: should be using these named strings): please update to include this one.